### ISSUE TITLE
--installDir option option not working properly

### ISSUE THREAD

**Issue by marvhh** [shopwareLabs/sw-cli-tools#61]
Hi,

I get following strange error:
`$ /opt/sw-cli-tools/bin/sw install:release -r latest -d xxx -i /path/to/shopware -p xxx --username xxx --password xxx --name Administrator --mail admin@admin.tld`

result:
```
 [RuntimeException]                                           
  Command failed. Error Output:                                
  checkdir:  cannot create extraction directory: path/to/shopware  
             No such file or directory
```

/path/to/shopware exists. It seems that the installer is cutting the leading "/".

Following command works properly
`cd /path/to/shopware && /opt/sw-cli-tools/bin/sw install:release -r latest -d xxx -i . -p xxx --username xxx --password xxx --name Administrator --mail admin@admin.tld`

**Comment by janbuecker** [comment]
I've already fixed that problem but I haven't committed it yet because I need to test it first. I'll keep you up2date-

**Comment by marvhh** [comment]
Nice to hear. If you want to attach the patch here, I can check it too.

**Comment by janbuecker** [comment]
Here you go ;) PR #62

**Comment by bcremer** [comment]
Closed as PR #62 was merged.
